src/test/test-locale-util.c

```diff
@@ -50,9 +50,38 @@ static void test_locale_is_valid(void) {
         assert_se(!locale_is_valid("\x01gar\x02 bage\x03"));
 }
 
+static void test_keymaps(void) {
+        _cleanup_strv_free_ char **kmaps = NULL;
+        char **p;
+        int r;
+
+        assert_se(!keymap_is_valid(""));
+        assert_se(!keymap_is_valid("/usr/bin/foo"));
+        assert_se(!keymap_is_valid("\x01gar\x02 bage\x03"));
+
+        r = get_keymaps(&kmaps);
+        if (r == -ENOENT)
+                return 0; /* skip test if no keymaps are installed */
```

Comment:
function returns void, but you return "0" here. That must have caused a compiler warning, no?

Reply:
semaphore shows the compiler warning, actually. Please make sure that your code compiles locally without warnings before submitting!

Reply:
I overlooked this when merging the 2 test functions sorry.
Fixed now.